src/v/cluster/tests/cluster_test_fixture.h

```diff
@@ -114,9 +118,12 @@ class cluster_test_fixture {
       int proxy_port_base = 8082,
       int schema_reg_port_base = 8081,
       int coproc_supervisor_port_base = 43189,
-      configure_node_id use_node_id = configure_node_id::yes) {
+      configure_node_id use_node_id = configure_node_id::yes,
+      empty_seed_starts_cluster empty_seed_starts_cluster_val
```

Comment:
nit: we should probably distinguish between setting the `empty_seed_starts_cluster` config and configuring with empty seeds.